ui/src/pages/backoffice/Patron/PatronDetails/components/PatronDocumentRequests/PatronDocumentRequests.js

```diff
@@ -30,24 +30,23 @@ export default class PatronDocumentRequests extends Component {
 
   viewDetails = ({ row }) => {
```

Comment:
I think this viewDetails code is repeated 3 or 4 times by now. Does it make sense to extract it?